internal/envoy/bootstrap.go

```diff
@@ -17,24 +17,26 @@
 package envoy
 
 import (
+	"fmt"
 	"os"
 
 	"github.com/golang/protobuf/jsonpb"
 	"github.com/golang/protobuf/proto"
+	"github.com/projectcontour/contour/pkg/config"
 )
 
 // SDSResourcesSubdirectory stores the subdirectory name where SDS path resources are stored to.
 const SDSResourcesSubdirectory = "sds"
 
 // SDSTLSCertificateFile stores the path to the SDS resource with Envoy's
 // client certificate and key for XDS gRPC connection.
-const SDSTLSCertificateFile = "xds-tls-certicate.json"
+const SDSTLSCertificateFile = "xds-tls-certificate.json"
```

Comment:
didn't look deeply into how this is used, but will changing this possibly break existing installs that expect the old typo'd name?

Reply:
I don't think so, but can also revert since it's not really related. Envoy should look at the directory not the file names.

Reply:
@skriss I rolled this back

Reply:
It should not break any existing installs. If the bootstrap version is updated, it will write and consume the new name. There's no case where an old bootstrap version would write the old name and a new version would try to consume the new name.